README.md

```diff
@@ -198,22 +198,28 @@ Puppeteer creates its own browser user profile which it **cleans up on every run
    displaying. Instead of launching in headless mode, launch a full version of
    the browser using  `headless: false`:
 
-        const browser = await puppeteer.launch({headless: false});
+```js
```

Comment:
Adding the backticks for syntax highlighting sounds great!
However, these backticked blocks must still be indented with 4 spaces so that they become part of the list item, instead of interrupting the list. Could you please make these changes?

Reply:
Good observation, fixed.